ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -611,9 +611,12 @@ computationExpr options { stubBase="FSharpExpressionBase"; }:
 
 lambdaExpr options { stubBase="FSharpExpressionBase"; }:
```

Comment:
Please add `Parameters` getter via `extras` block, so it's easier to work with them and existing code doesn't need any changes.